### ISSUE TITLE
Where do we provide overall info about connector resources

### ISSUE THREAD

**Issue by clcarson** [data-8/connector-instructors#41]
Currently in 
https://docs.google.com/spreadsheets/d/1gcJ_zkvkbVzOu1z1HFtIKRAfczcA5QaELNBkvcTwRLM/edit?ts=587e91b6#gid=0

**Comment by choldgraf** [comment]
wow that's a great doc - I'd never seen it before. We could easily link to it from the wiki, or if you think it should be stored on the wiki itself, perhaps in the `contacts` page? And maybe with a link on the home page as well?

**Comment by clcarson** [comment]
Agreed, waiting to finalize it with @anthonysuen and his team. This is tied into where we put the Connector Assistant info.

**Comment by anthonysuen** [comment]
I agree that we should store this on the wiki contact page and somewhere on the main page too. 

Anthony

**Comment by choldgraf** [comment]
@anthonysuen are you + your team transferring this information (or at least the link) to the wiki?

@gunjanbaid if this is something you've changed, ping us in this (or other) issues so that we know to close them!

**Comment by gunjanbaid** [comment]
I updated the contacts page to refer instructors to their CAs and the Piazza and removed individual staff emails. I can link this document on that page as well, if that's desired.

**Comment by choldgraf** [comment]
Sounds good. @clcarson what do you think?
